src/test/kotlin/TestWorld.kt

```diff
@@ -0,0 +1,32 @@
+import io.hsar.wh40k.combatsimulator.model.MapPosition
+import io.hsar.wh40k.combatsimulator.model.UnitInstance
+import io.hsar.wh40k.combatsimulator.model.World
+import io.hsar.wh40k.combatsimulator.model.unit.*
+import org.junit.Test
+import org.junit.jupiter.api.TestInstance
+import kotlin.test.assertEquals
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class TestWorld {
+
+    @Test
+    fun testDistanceApart() {
+        val world = TestUtils.getGenericTwoUnitWorld(MapPosition(1,6), MapPosition(10, 4))
+        assertEquals(9,world.distanceApart(world.friendlyForces[0], world.enemyForces[0]))
```

Comment:
Ah, I hate to do this to you because you're already using a new language, but there is a standard test library called Hamcrest (or Google-colloquial "assertThat") that is in the POM that I would be expecting our tests to use. 

I'm going to push a commit to develop with an example test that will hopefully show you what's going on.

Reply:
See commit fcb39fa3

Reply:
I'll have a look and update the branch to use Hamcrest